app/views/steps/case/date_stamp/show.html.erb

```diff
@@ -0,0 +1,18 @@
+<% title t('.page_title') %>
+<% step_header %>
+
+<div class="govuk-grid-row app-inverted--card app-inverted-text">
+  <div class="govuk-grid-column-full">
+    <h1 class="govuk-heading-l app-inverted-text">Your application has been date stamped</h1>
```

Comment:
We can either convert this view to a localised view (adding extension `.en.html.erb`) or extract this string to locales.
I think as this page is so minimal regarding content, probably best just to extract to locales as all the other copy in the page is already in locales.

Reply:
I think i'll just add it to the locals